<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="15" column="4">
     <widget class="QLineEdit" name="conn_counter_7"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="server_IP"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="start_conns_stress">
      <property name="text">
       <string>start stress</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="btn_connect_2">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="conn_status_1"/>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QLineEdit" name="conn_status_3"/>
    </item>
    <item row="9" column="4">
     <widget class="QLineEdit" name="conn_counter_4"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>server PORT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>CONNECTION 4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="btn_connect_3">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLineEdit" name="conn_status_2"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>server IP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QPushButton" name="btn_connect_5">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="15" column="5">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QLineEdit" name="conn_counter_6"/>
    </item>
    <item row="13" column="6">
     <widget class="QLineEdit" name="conn_status_6"/>
    </item>
    <item row="11" column="6">
     <widget class="QLineEdit" name="conn_status_5"/>
    </item>
    <item row="15" column="8">
     <widget class="QPushButton" name="btn_disconnect_7">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="9" column="8">
     <widget class="QPushButton" name="btn_disconnect_4">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="15" column="6">
     <widget class="QLineEdit" name="conn_status_7"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="conn_counter_1"/>
    </item>
    <item row="13" column="5">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CONNECTION 2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>CONNECTION 3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CONNECTION 1</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QPushButton" name="btn_connect_4">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QLineEdit" name="conn_status_4"/>
    </item>
    <item row="11" column="8">
     <widget class="QPushButton" name="btn_disconnect_5">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="btn_disconnect_2">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>log_info</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="15" column="7">
     <widget class="QPushButton" name="btn_connect_7">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="btn_disconnect_1">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="7" column="8">
     <widget class="QPushButton" name="btn_disconnect_3">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="8">
     <widget class="QPushButton" name="btn_disconnect_6">
      <property name="text">
       <string>disconnect</string>
      </property>
     </widget>
    </item>
    <item row="13" column="7">
     <widget class="QPushButton" name="btn_connect_6">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLineEdit" name="conn_counter_3"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="btn_connect_1">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4" colspan="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>CONNECTION 5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>status</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="server_PORT"/>
    </item>
    <item row="12" column="4" colspan="3">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>CONNECTION 6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>counter</string>
      </property>
     </widget>
    </item>
    <item row="14" column="4" colspan="3">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>CONNECTION 7</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QLineEdit" name="conn_counter_5"/>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="conn_counter_2"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="active_conns"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stop_conns_stress">
      <property name="text">
       <string>stop stress</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>active conns</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>result</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="conns_stress_result"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="log_info_3">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="log_info_2">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="log_info_4">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="log_info_5">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="log_info_6">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="log_info_7">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="log_info_1">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
